### PR TITLE
Support ruby 2.6

### PR DESCRIPTION
**Pull request by Paxa**
Test more rubies

### CODE REVIEW

**Comment by Paxa** [comment]
Tests fail because latest bundler doesn't support ruby 2.2 and ruby 2.1
For 2.3 - 2.6 works well

**Comment by iczman** [comment]
From TravisCI:

```
$ gem update --system
Updating rubygems-update
Fetching: rubygems-update-3.0.2.gem (100%)
ERROR:  Error installing rubygems-update:
	rubygems-update requires Ruby version >= 2.3.0.
ERROR:  While executing gem ... (NoMethodError)
    undefined method `version' for nil:NilClass
The command "gem update --system" failed and exited with 1 during .
```

rubygems 3 doesn't work on Ruby < 2.3.0. There are two ways that I see to resolve this:
- install rubygems 2.x like [this](https://github.com/rubygems/rubygems/issues/2534#issuecomment-448843746), or
- stay on rubygems 3 and drop testing and maybe lookse's support for Ruby 2.1 & 2.2

@oggy which one do you prefer?

**Comment by zw963** [comment]
I think we can use solution 2,  just release a new version looksee, which drop support Ruby 2.1 & 2.2.

**Comment by Paxa** [comment]
It still should support 2.1 and 2.2 just hard to test it in CI

**Comment by zw963** [comment]
@Paxa  How is it going now? Thanks

**Comment by Paxa** [comment]
travis can run tests now, but somehow failing for ruby 2.2

You already can use it:
```ruby
gem 'looksee', github: 'paxa/looksee'
```

**Comment by zw963** [comment]
@Paxa , It worked! Thanks

**Comment by iczman** [comment]
@oggy any idea how to resolve the test failure on 2.2?

**Comment by mathieujobin** [comment]
@Paxa can you put back 2.2.5 and maybe all revision in between up to 2.2.10, to see it might be a problem starting from one of those revision.

**Comment by mathieujobin** [comment]
Confirmed. It pass up to 2.2.9. Fails only for 2.2.10

https://travis-ci.org/mathieujobin/looksee/builds/501157321

**Comment by Paxa** [comment]
Thanks, applied

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,15 @@
 language: ruby
 before_install:
-  - gem update --system
-  - gem install bundler
+#  - gem update --system
```
Comment (mathieujobin) [inline .travis.yml]: can you remove this commented out line please?

Comment (mathieujobin) [reply .travis.yml]: maybe squash it and the first two commits into one ... or everything into one commit

Comment (Paxa) [reply .travis.yml]: Done

**Comment by oggy** [comment]
Thanks @Paxa, and everyone else!

To answer the question above, I'm happy to drop support for the older versions. The ruby world is considerably different to when this gem first started out, and following the ruby support cycle these days seems more appropriate.

**Pull request merged**